### DISCUSSION TITLE
JS Object returned from create function is missing from db

### DISCUSSION THREAD

**Discussion by jsoneaday** [prisma/prisma#9477] [Q&A] [answered]
When I run this function below I get back a js object with the appropriate fields. However when I query the db for the insert entity it is not in the table. Is there a delay of some kind before Prisma does a write?

```
export async function createResponseMessage(
  prisma: PrismaClient,
  content: string,
  authorId: bigint,
  respondingToId: bigint,
  { transactionId, transWalletAddr, signature }: TransactionParams
): Promise<bigint | null> {
  try {
    const message = await prisma.message.create({
      data: {
        content,
        transaction: {
          create: { transactionId, transWalletAddr, signature },
        },
        author: {
          connect: { id: authorId },
        },
        respondingTo: {
          connect: { id: respondingToId },
        },
      },
    });

    return message.id;
  } catch (e) {
    throw e;
  }
  return null;
}
```

**Comment by jsoneaday** [comment, marked as answer]
This is an error in my code. How can I delete this entry?

**Reply by ryands17** [reply]
Just marking it answered as you've solved it yourself.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
